### PR TITLE
Update camranger.rb

### PR DESCRIPTION
**Pull request by leoj3n**
Remove version key from app location

### CODE REVIEW

File: Casks/camranger.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
   url "http://www.camranger.com/downloadFiles/CamRanger_#{version.gsub('.', '_')}.dmg"
   name 'CamRanger'
-  homepage 'http://www.camranger.com/downloads/#fbox_5594'
+  homepage 'http://camranger.com/mac-downloads'
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
 
-  app "#{version}/CamRanger.app"
+  app "CamRanger.app"
```
Comment (vitorgalvao) [inline Casks/camranger.rb]: dquotes

Comment (vitorgalvao) [inline Casks/camranger.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by leoj3n** [comment]
Ok @vitorgalvao should be ready now but I did it through the web interface so don't think I can squash without downloading the patch branch locally which I'm not able to do right now.

**Comment by vitorgalvao** [comment]
No problem, I’ll pull and squash, then.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3f3def7a15ff002e48707d469d5f3cf83e2c9db9. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**